x-pack/docs/en/security/authentication/saml-guide.asciidoc

```diff
@@ -505,11 +505,11 @@ support. Since PEM format is the most commonly supported format, the examples
 below will generate certificates in that format.
 
 Using the <<certutil,`elasticsearch-certutil` tool>>, you can generate a
-signing certificate with the following command:
+self-signed certificate with the following command:
```

Comment:
The most important characteristic is that this is a `signing` certificate (used to sign SAML messages), not that it is a `self-signed` one. I'd suggest we revert this change

Reply:
You are right. Reverted.